packages/pluggable-widgets-tools/configs/rollup.config.native.js

```diff
@@ -35,6 +35,36 @@ const outWidgetFile = join(widgetPackage.replace(/\./g, "/"), widgetName.toLower
 const mpkDir = join(sourcePath, "dist", widgetVersion);
 const mpkFile = join(mpkDir, process.env.MPKOUTPUT ? process.env.MPKOUTPUT : `${widgetPackage}.${widgetName}.mpk`);
 
+const extensions = [".js", ".jsx", ".tsx", ".ts"];
+
+const editorConfigExternal = [
+    // "mendix" and internals under "mendix/"
+    /^mendix($|\/)/,
+
+    // "react"
+    /^react$/,
+
+    // "react/jsx-runtime"
+    /^react\/jsx-runtime$/,
+
+    // "react-dom"
+    /^react-dom$/
+];
+
+const nativeExternal = [
+    /^mendix($|\/)/,
+    /^react-native($|\/)/,
+    /^big.js$/,
+    /^react($|\/)/,
+    /^react-native-gesture-handler($|\/)/,
+    /^react-native-reanimated($|\/)/,
+    /^react-native-fast-image($|\/)/,
+    /^react-native-svg($|\/)/,
+    /^react-native-vector-icons($|\/)/,
+    /^react-navigation($|\/)/,
+    /^react-native-safe-area-context($|\/)/
+];
+
```

Comment:
In this file we seem to only have a change of location for those constants. What is the purpose of it?

Reply:
The purpose is to just organize better those variables.

Even though it works in this case to have const variables at the bottom of the file, it is very strange since const variables are not hoisted and the only reason for this to work is because the function is called at a later stage.

Reply:
I'm extending this change and moving the other variables also inside this file to the top.

Reply:
Fair enough. 👍